source/GooglePlayServicesProject.cshtml

```diff
@@ -252,9 +252,13 @@
       }
     }
     <!-- Pull in the play core extensions helpers -->
-    @if (@Model.NuGetPackageId == "Xamarin.Google.Android.Play.Core")
```

Comment:
Is removing this going to break the `Xamarin.Google.Android.Play.Core` package?

Reply:
Good point. I had forgotten that this package was still going to be available (even though you can't use it on API 34). I'll put the old API's back. 

This does mean however if someone has both `Xamarin.Google.Android.Play.Core` and `Xamarin.Google.Android.Play.Asset.Delivery` in their project they WILL get a build error because the Wrapper classes are identical, since both assemblies contain the same classes. Google just moved the asset.delivery and splitinstall classes to a new library, they did not rename them or change the package names. 

For example `com.google.android.play.core.splitinstall.SplitInstallManager` still exists in the `com.google.android.play:feature-delivery` as well as the `com.google.android.play:core` package.